test/core/channel/channelz_test.cc

```diff
@@ -449,16 +449,16 @@ TEST_F(ChannelzRegistryBasedTest, GetTopChannelsMoreGaps) {
   Json array = (*parsed_json->mutable_object())["channel"];
   ValidateJsonArraySize(array, 2);
   std::vector<intptr_t> uuids = GetUuidListFromArray(array.array_value());
-  EXPECT_EQ(static_cast<intptr_t>(3), uuids[0]);
-  EXPECT_EQ(static_cast<intptr_t>(5), uuids[1]);
+  EXPECT_EQ(intptr_t{3}, uuids[0]);
```

Comment:
In all of the cases where we're specifying a type inside of a gtest `EXPECT_*` macro, it's generally better to just use `U` or `UL` suffixes instead of spelling out the specific type.  So for example, in this case, you can change `intptr_t{3}` to `3UL`.  Or if a test currently says `unsigned{1}`, you can change that to `1U`.

Please fix throughout.

Reply:
Is 3UL guaranteed to be intptr_t on all platforms? (I genuinely don't know)

Reply:
Good question.  I know that I've always used a `UL` suffix in `EXPECT_*` macros for things like `size_t`, and it's always worked fine, so I assume it will for `intptr_t` too.  But if it causes the test to fail on any platform we care about, we can always switch back to the current syntax for any `intptr_t` values.

Reply:
`EXPECT_EQ` doesn't require the type of two given value to have the exact same so I think it should be fine with just plain 3 here.

Reply:
Historically, I have found these suffixes to frequently be needed to get `EXPECT_*` macros to compile properly on all platforms.  But if it works without it, that's fine with me.